openpype/hosts/houdini/plugins/publish/validate_output_node.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import (
+    SelectInvalidAction,
+    SelectROPAction,
+)
+
+import hou
+
+
+class ValidateOutputNode(pyblish.api.InstancePlugin):
+    """Validate the instance Output Node.
+
+    This will ensure:
+        - The Output Node Path is set.
+        - The Output Node Path refers to an existing object.
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["fbx"]
+    hosts = ["houdini"]
+    label = "Validate Output Node"
+    actions = [SelectROPAction, SelectInvalidAction]
+
+    def process(self, instance):
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            raise PublishValidationError(
+                "Output node(s) are incorrect",
```

Comment:
The output node is never supposed to be plural, right? Or can it?

Reply:
it can, each object node child with its sop output when `startnode` is a subnetwork.